NEWS

```diff
@@ -63,6 +63,14 @@ CHANGES WITH 240 in spe:
           has been added to revert this change in behaviour, which might be
           an option for systems that turn off memcg in the kernel.
 
+        * When no /etc/locale.conf file exists (and hence no locale settings
+          are in place), systemd will now use the "C.UTF-8" locale by default,
+          and set LANG= to it. This locale has been supported by various
+          distributions already and recently Fedora started to support it too,
+          with clear indications that upstream glibc is going to make it
+          available soon too. This locale enables UTF-8 mode by default, which
```

Comment:
Let's not say "soon" here. Upstream is working on support, but as with all software, there's no ETA.